packages/legacy-client/src/client/objects/convertWireToOsdkObject.test.ts

```diff
@@ -127,6 +127,34 @@ describe("convertWireToOsdkObject", () => {
       JSON.stringify(wireObject, null, 2),
     );
   });
+
+  it("handles reserved word backcompat in property and link names", () => {
+    const wireObject = {
+      catch: 1,
+      __primaryKey: 1,
+      __apiName: "ObjectTypeWithReservedNames",
+      __rid: "rid.1",
+    } as const;
+    const object = convertWireToOsdkObject<
+      "ObjectTypeWithReservedNames",
+      typeof MockOntology
+    >(client, wireObject);
+
+    expect(object.catch).toEqual(1);
+    expect((object as any).catch_).toEqual(1);
+
+    expect((object as any).const_.get).toBeTypeOf("function");
+
+    expect(object.toString()).toMatchInlineSnapshot(`
+      "{
+        \\"catch\\": 1,
+        \\"catch_\\": 1,
+        \\"__primaryKey\\": 1,
+        \\"__apiName\\": \\"ObjectTypeWithReservedNames\\",
+        \\"__rid\\": \\"rid.1\\"
+      }"
+    `);
```

Comment:
I think we need to figure out what to do about this. Should we consider it a break as there are additional things in the output than before?

Reply:
Ok we'll use the suffixed names and NOT add the unsuffixed ones for now.